### PR TITLE
More strings fixed

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: app/src/main/res/values/strings.xml
Comment (MuntashirAkon) [inline app/src/main/res/values/strings.xml:526]: I think this needs some rework, something like _Could not install app because another app has this package name._

File: app/src/main/res/values/strings.xml
Comment (MuntashirAkon) [inline app/src/main/res/values/strings.xml:646]: Preference items do not have a full-stop at the end. If you're going to add a full-stop to one item, you have to add them to all of them.

File: app/src/main/res/values/strings.xml
Comment (MuntashirAkon) [inline app/src/main/res/values/strings.xml:676]: It's incorrect to add a comma before _as_.

File: app/src/main/res/values/strings.xml
Comment (MuntashirAkon) [inline app/src/main/res/values/strings.xml:680]: I think this needs some rework, something like _Could not install app because a system app has this package name with a different signature._

File: app/src/main/res/values/strings.xml
Comment (MuntashirAkon) [inline app/src/main/res/values/strings.xml:689]: I think the modification has destroyed the original meaning. There's no option to send the signatures in the dialog which will appear after clicking here. However, there is an option to copy the signatures, and the user can send them to me manually via other means. (Probably I should change this as well?)

File: app/src/main/res/values/strings.xml
Comment (MuntashirAkon) [inline app/src/main/res/values/strings.xml:700]: No space between _can_ and _not_.

File: app/src/main/res/values/strings.xml
Comment (MuntashirAkon) [inline app/src/main/res/values/strings.xml:705]: Again, preference items do not have a full-stop at the end. If you're going to add a full-stop to one item, you have to add them to all of them.

**Review by MuntashirAkon — CHANGES_REQUESTED** [review]
I've addressed some issues that need to be fixed before I can merge.

File: app/src/main/res/values/strings.xml
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:676]: It is correct here though https://magnumproofreading.com/blog/proper-punctuation/comma-before-as/

File: app/src/main/res/values/strings.xml
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:131]: Is that because there already is a "APK Parser?

File: app/src/main/res/values/strings.xml
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:689]: I can leave it like this, and you can add a "send via" dialog?

File: app/src/main/res/values/strings.xml
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:705]: There is one for the first sentence, so it follows it should be there in the second one. Will add for all.

File: app/src/main/res/values/strings.xml
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:646]: Did for all preference messages

File: app/src/main/res/values/strings.xml
Comment (MuntashirAkon) [reply app/src/main/res/values/strings.xml]: It seems the developer himself/herself is confused about the library name. Besides, this is only a temporary solution. I'll move to a better solution in future.

File: app/src/main/res/values/strings.xml
Comment (MuntashirAkon) [reply app/src/main/res/values/strings.xml:705]: Thanks.

File: app/src/main/res/values/strings.xml
Comment (MuntashirAkon) [inline app/src/main/res/values/strings.xml:519]: The change doesn't seem to reflect the original meaning (the original meaning is also somewhat incorrect though). There are three ways a component can be configured: enable, disable and default (block/unblock terminologies are widely used in privacy communities though). When a component is unblocked, it isn't enabled by AM rather it is set to default (Android figures out what to do with the component). But AM only unblocks the components that are _configured_ within AM. If you block a component using MAT, for example, but not on AM, the component won't be enabled.

File: app/src/main/res/values/strings.xml
Comment (MuntashirAkon) [inline app/src/main/res/values/strings.xml:261]: _enabling_ should be removed :)

**Review by MuntashirAkon — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml
Comment (MuntashirAkon) [reply app/src/main/res/values/strings.xml:689]: Alright. I'll replace _copy_ with _send_.

File: app/src/main/res/values/strings.xml
Comment (MuntashirAkon) [inline app/src/main/res/values/strings.xml:519]: This is still problematic. Currently, there are three things that are reverted when using this operation: app components (not the app itself), app ops and permissions.
1. App components (not apps themselves) are set to their _default_ state
2. App ops are set to their _default_ values
3. Permissions will be granted (and not set to their default values)

**Review by MuntashirAkon — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml
Comment (MuntashirAkon) [reply app/src/main/res/values/strings.xml:519]: Oh, you have to escape the quotations. So, `'` should be `\'`.

Comment (comradekingu) [inline app/src/main/res/values/strings.xml:519]:
```suggestion
    <string name="pref_remove_all_rules_msg">Permissions are granted, app ops and components will return to default values.</string>
```

**Review by MuntashirAkon — APPROVED** [review]

**Pull request merged**
